solutions/observability/logs/stream-any-log-file-using-edot-collector.md

```diff
@@ -0,0 +1,374 @@
+tar xzvf elastic-agent-{{version.stack}}-darwin-x86_64.tar.gz
+```
+:::
+
+:::{tab-item} Windows
+
+```powershell subs=true
+# PowerShell 5.0+
+wget https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-{{version.stack}}-windows-x86_64.zip -OutFile elastic-agent-{{version.stack}}-windows-x86_64.zip
+Expand-Archive .\elastic-agent-{{version.stack}}-windows-x86_64.zip
+```
+:::
+
+::::
+:::::
+
+:::::{step} Configure the EDOT Collector
+
+Follow these steps to retrieve the managed OTLP endpoint URL for your Serverless project:
+
+1. In Elastic Cloud Serverless, open your Observability project.
+2. Go to **Add data** → **Application** → **OpenTelemetry**.
+3. Select **Managed OTLP Endpoint** in the second step.
+4. Copy the OTLP endpoint configuration value.
+5. Select **Create API Key** to generate an API key.
+
+Replace `<ELASTIC_OTLP_ENDPOINT>` and `<ELASTIC_API_KEY>` before applying the following commands:
+
+::::{tab-set}
+
+:::{tab-item} Linux
+
+```bash
+ELASTIC_OTLP_ENDPOINT=<ELASTIC_OTLP_ENDPOINT> && \
+ELASTIC_API_KEY=<ELASTIC_API_KEY> && \
+cp ./otel_samples/managed_otlp/logs_metrics_traces.yml ./otel.yml && \
+mkdir -p ./data/otelcol && \
+sed -i "s#\${env:STORAGE_DIR}#${PWD}/data/otelcol#g" ./otel.yml && \
+sed -i "s#\${env:ELASTIC_OTLP_ENDPOINT}#${ELASTIC_OTLP_ENDPOINT}#g" ./otel.yml && \
+sed -i "s#\${env:ELASTIC_API_KEY}#${ELASTIC_API_KEY}#g" ./otel.yml
+```
+:::
+
+:::{tab-item} macOS
+
+```bash
+ELASTIC_OTLP_ENDPOINT=<ELASTIC_OTLP_ENDPOINT> && \
+ELASTIC_API_KEY=<ELASTIC_API_KEY> && \
+cp ./otel_samples/managed_otlp/logs_metrics_traces.yml ./otel.yml && \
+mkdir -p ./data/otelcol && \
+sed -i '' "s#\${env:STORAGE_DIR}#${PWD}/data/otelcol#g" ./otel.yml && \
+sed -i '' "s#\${env:ELASTIC_OTLP_ENDPOINT}#${ELASTIC_OTLP_ENDPOINT}#g" ./otel.yml && \
+sed -i '' "s#\${env:ELASTIC_API_KEY}#${ELASTIC_API_KEY}#g" ./otel.yml
+```
+:::
+
+:::{tab-item} Windows
+
+```powershell
+Remove-Item -Path .\otel.yml -ErrorAction SilentlyContinue
+Copy-Item .\otel_samples\managed_otlp\logs_metrics_traces.yml .\otel.yml
+New-Item -ItemType Directory -Force -Path .\data\otelcol | Out-Null
+
+$content = Get-Content .\otel.yml
+$content = $content -replace '\${env:STORAGE_DIR}', "$PWD\data\otelcol"
+$content = $content -replace '\${env:ELASTIC_OTLP_ENDPOINT}', "<ELASTIC_OTLP_ENDPOINT>"
+$content = $content -replace '\${env:ELASTIC_API_KEY}', "<ELASTIC_API_KEY>"
+$content | Set-Content .\otel.yml
+```
+:::
+::::
+:::::
+
+:::::{step} Configure log file collection
+
+To collect logs from specific log files, you need to modify the `otel.yml` configuration file. The configuration includes receivers, processors, and exporters that handle log data.
+
+::::{tab-set}
+:group: stack-serverless
+
+:::{tab-item} Elastic Stack
+:sync: stack
+
+Here's an example configuration for collecting log files with Elastic Stack:
+
+:::{dropdown} otel.yml for logs collection (Elastic Stack)
```

Comment:
The guide seems to be serverless specific, do we want the stack yml as well?

Reply:
The name of the file doesn't matter for the Collector, but it's usually otel.yml by convention